app/assets/stylesheets/app/sortable.scss

```diff
@@ -1,19 +1,19 @@
 .sortable {
-  background-image: image-url("icon-sort.gif");
+  background-image: image-url("icon-sort.gif"); #
```

Comment:
Honestly, I have no idea. Is font-awesome now all SVG or fonts? If we can't figure it out, maybe we just pull these files out of the old version of font-awesome and include them in nucore.